### PR TITLE
Add a link target for "chapter 4"

### PR DESCRIPTION
**Pull request by 126016**
Add a link target for "chapter 4"

### CODE REVIEW

File: Documentation/3-BlogExample/Index.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ background. This knowledge will then help you in the creation of your own
 extension.
 
 If you are looking for a manual specifically on creating an extension,
-chapter 4 will show you the right set of tools. However, we recommend to
+:ref:`chapter 4 <creating-a-first-extension>` will show you the right set of tools. However, we recommend to
```
Comment (mbrodala) [inline Documentation/3-BlogExample/Index.rst:26]: Since you mentioned this in #347: please use this instead: 

```
:ref:`creating-a-first-extension`
```

Comment (126016) [reply Documentation/3-BlogExample/Index.rst:26]: Okay 
:ref:`creating-a-first-extension` is the syntax for the link, but where come the text for the link "chapter 4" ?

Comment (126016) [reply Documentation/3-BlogExample/Index.rst:26]: I found this documentation:
https://www.sphinx-doc.org/en/1.7/markup/inline.html#cross-referencing-arbitrary-locations

And it says that "Labels that aren’t placed before a section title can still be referenced, but you must give the link an explicit title, using this syntax: :ref:`Link title <label-name>`." 

So I used that. Not quite sure I am doing this for the first time.

Comment (mbrodala) [reply Documentation/3-BlogExample/Index.rst:26]: It is inserted automatically, see https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/CheatSheet.html#cheat-sheet-intersphinx

Comment (126016) [reply Documentation/3-BlogExample/Index.rst]: I change it in :ref:`creating-a-first-extension`

**Comment by mbrodala** [comment]
@126016 the change has been [deployed now](https://docs.typo3.org/m/typo3/book-extbasefluid/master/en-us/3-BlogExample/Index.html):

image

**Pull request merged**